src/Features/LanguageServer/Protocol/Features/Options/CompletionOptionsStorage.cs

```diff
@@ -110,6 +110,6 @@ public static CompletionOptions GetCompletionOptions(this IGlobalOptionService o
             storageLocation: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Specific.ProvideDateAndTimeCompletions"));
 
     public static readonly PerLanguageOption2<bool?> ShowNewSnippetExperience
-        = new(nameof(CompletionOptions), nameof(ShowNewSnippetExperience), CompletionOptions.Default.ShowNewSnippetExperience,
+        = new(nameof(CompletionOptions), nameof(ShowNewSnippetExperience), CompletionOptions.Default.ShowNewSnippetExperienceUserOption,
```

Comment:
Do we need to change the string for storage location to completely avoid any conflcit from previous persisted value?

Reply:
No, we wanted to keep it as is since we did not want to override users changes.